CONTRIBUTING.md

```diff
@@ -72,3 +72,60 @@ A PR is considered to be **ready to merge** when:
 * Urgent fixes can take exceptions as long as it has been actively communicated.
 
 Any Approver / Maintainer can merge the PR once it is **ready to merge**.
+
+## Build and Run Code Examples
```

Comment:
I would suggest putting this section under `Development` (above `How to Send Pull Requests`).